sensors_description/urdfs/sensors/realsense_d455.gazebo.xacro

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
+
+<!--
+License: Apache 2.0. See LICENSE file in root directory.
+Copyright(c) 2017 PAL Robotics, S.L. All Rights Reserved
+
+This is the Gazebo URDF model for the Intel RealSense D435 camera
+-->
+
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <xacro:macro name="gazebo_d455" params="camera_name reference_link topics_ns 
+  depth_optical_frame color_optical_frame infrared1_optical_frame infrared2_optical_frame" >
+
+    <!-- Load parameters to model's main link-->
+    <xacro:property name="deg_to_rad" value="0.01745329251994329577" />
+    <gazebo reference="${reference_link}">
+      <self_collide>0</self_collide>
+      <enable_wind>0</enable_wind>
+      <kinematic>0</kinematic>
+      <mu2>1</mu2>
+      <fdir1>0 0 0</fdir1>
+      <kp>1e+13</kp>
+      <kd>1</kd>
+
+      <sensor name="${camera_name}color" type="camera">
```

Comment:
Is it normal that there is no underscore between camera_name and color? (same q. for the other lines below)

Reply:
This was the same convention used in the old d435 gazebo plugin urdf made in PAL. I left it in this way because that's only the name of the sensor and it is not used neither for the tfs nor for the topics.